apps/total-typescript/src/templates/workshop-template.tsx

```diff
@@ -267,6 +268,12 @@ const LessonListItem = ({
     module: workshop,
   })
 
+  const moduleProgress = trpc.moduleProgress.bySlug.useQuery({
+    slug: workshop.slug.current,
+  })
+
+  console.log({moduleProgress})
```

Comment:
this was just a spot to drop it in and check it out

Reply:
I'm gonna drop a PR testing it out!